interfaces/apparmor/template.go

```diff
@@ -885,6 +885,17 @@ deny ptrace (trace),
 deny capability sys_ptrace,
 `
 
+var sysModuleCapabilityDenySnippet = `
+# The rtnetlink kernel interface can trigger the loading of kernel modules,
+# first attempting to operate on a network module (this requires the net_admin
+# capability) and falling back to loading ordinary modules (and this requires
+# the sys_module capability). For reference, see the dev_load() function in:
+# https://kernel.ubuntu.com/git/ubuntu/ubuntu-focal.git/tree/net/core/dev_ioctl.c#n332
```

Comment:
This should probably reference tree state for an actual entry in git log, as HEAD may change and the line reference may become outdated, eg. https://kernel.ubuntu.com/git/ubuntu/ubuntu-focal.git/tree/net/core/dev_ioctl.c?id=43b79d50559b25340663bad89b6f33a6fd2a1451#n332